Sources/SwifterSwift/CoreAnimation/CATransform3DExtensions.swift

```diff
@@ -0,0 +1,261 @@
+//
+//  CATransform3DExtensions.swift
+//  SwifterSwift
+//
+//  Created by Guy Kogus on 19/3/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(QuartzCore)
+
+import QuartzCore
+
+// MARK: - Equatable
+extension CATransform3D: Equatable {
+
+    // swiftlint:disable missing_swifterswift_prefix
```

Comment:
Why disable this here?

Reply:
Cos it's an exact copy of the Swift documentation, it's not a custom function.